kubetest/azure.go

```diff
@@ -769,6 +775,31 @@ func (c *Cluster) createCluster() error {
 
 }
 
+func (c *Cluster) populateAzureCloudConfig() error {
+	cloudConfig, err := json.MarshalIndent(map[string]string{
+		"cloud":           c.azureEnvironment,
+		"tenantId":        c.credentials.TenantID,
+		"subscriptionId":  c.credentials.SubscriptionID,
+		"aadClientId":     c.credentials.ClientID,
+		"aadClientSecret": c.credentials.ClientSecret,
+		"resourceGroup":   c.resourceGroup,
+		"location":        c.location,
```

Comment:
could we also add `vmType` to the cloudConfig? It would be required to determine wether a node is VM or VMSSVM. If it is hard to determine the vm type, we could use "vmss" as it supports both VM and VMSSVM.

Reply:
PTAL at the PR. Is there any other fields that I should add to the cloud config?

Reply:
thanks for the update, I think that's enough